.github/vale/styles/OpenSearch/MergeConflicts.yml

```diff
@@ -0,0 +1,6 @@
+extends: existence
```

Comment:
This rule is not needed because it's specific to leaving merge conflicts in documentation.